jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/SessionManager.java

```diff
@@ -316,12 +308,15 @@ protected boolean access(long accessTime) {
       if (dirty) {
       } else if (timeRemaining < (getSessionExpirationInMilliseconds() * UPDATE_TIMESTAMP_RATIO)) {
         dirty = true;
-        logger.fine("Session " + getId() + " accessed while near expiration, marking dirty.");
-      } else {
-        logger.fine("Session " + getId() + " accessed early, not marking dirty.");
+        if (logger.isLoggable(Level.FINE)) {
+          logger.fine(
+              String.format("Session %s accessed while near expiration, marking dirty.", getId()));
+        }
+      } else if (logger.isLoggable(Level.FINE)) {
+        logger.fine(String.format("Session %s accessed early, not marking dirty.", getId()));
       }
-      sessionData.setExpirationTime(System.currentTimeMillis()
-              + getSessionExpirationInMilliseconds());
+      sessionData
```

Comment:
I am surprised to this that the formatter did that... I don't think that is desired but fine with as is.